tests/regression/error/test-func-sliced.lus

```diff
@@ -48,4 +53,11 @@ function exp (x, y: real) returns (out: real);
    guarantee out >= id(out);
    guarantee out <= sin(out);
 *)
+let
+out = x + y;
+tel
+
 function sin(in: real) returns (out: int);
+let
+   out = (int in);
+tel
```

Comment:
Remove this body and set the function as imported:
function **imported** sin(in: real) returns (out: int);